libs/langchain/langchain/retrievers/chaindesk.py

```diff
@@ -13,17 +13,17 @@
 class ChaindeskRetriever(BaseRetriever):
     """`Chaindesk API` retriever."""
 
-    datastore_url: str
+    datastore_id: str
```

Comment:
@gmpetrov thanks for the fix. The is a breaking change for the API. Could you explain why this change is required (why is most of the URL hard-coded in the code?) and perhaps find a way to do it in a backwards comaptible way?